README.md

```diff
@@ -168,7 +168,6 @@ Table of contents:
 * [Nir kaufman](https://twitter.com/nirkaufman)
 * [Dmitriy Shekhovtsov](https://twitter.com/valorkin)
 * [Jeff Delaney](https://twitter.com/jeffdelaney23)
-* [**{{** add_expert **}}**](https://github.com/gdi2290/awesome-angular/edit/gh-pages/README.md)
```

Comment:
Why was this removed?

Reply:
The build failed and the reason was that this file wasn't available .

Reply:
The build failed, because this file wasn't available .

Reply:
There's probably lots of broken links. You can add it to the whitelist to ignore it for now.

Reply:
Aha , Thanks for the info

Reply:
Should I do something else?
Would you please explain how to add broken links to whitelist to ignore?

Reply:
See [validate_urls.sh](https://github.com/PatrickJS/awesome-angular/blob/gh-pages/scripts/validate_urls.sh).